packages/zoe/src/contractFacet/zcfMint.js

```diff
@@ -18,14 +18,14 @@ import './types.js';
 
 const { Fail } = assert;
 
-export const makeZCFMintFactory = async (
-  zcfBaggage,
-  recordIssuer,
-  getAssetKindByBrand,
-  makeEmptySeatKit,
-  reallocator,
+export const makeZCFMintFactory = (
+  /** @type {import('@agoric/vat-data').Baggage} */ zcfBaggage,
+  /** @type {{ (keyword: string, issuerRecord: IssuerRecord): void }} */ recordIssuer,
+  /** @type {GetAssetKindByBrand} */ getAssetKindByBrand,
+  /** @type { (exit?: undefined) => { zcfSeat: any; userSeat: Promise<UserSeat> }} */ makeEmptySeatKit,
+  /** @type {ZcfMintReallocator} */ reallocator,
```

Comment:
This seems like a radical shift in style from `@param`. Why? At least at first, I do not find it more readable.

Reply:
It's the style that "infer params from usage" automates. I think it's worth allowing because it removes the need to repeat the parameter name.